opal/mca/accelerator/base/help-accelerator-base.txt

```diff
@@ -0,0 +1,14 @@
+# -*- text -*-
+#
+# Copyright (c) 2022      Amazon.com, Inc. or its affiliates.
+#                         All Rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+[Multiple Accelerators Found]
+ERROR: Both %s and %s accelerator frameworks were successfully initialized,
```

Comment:
There can only be two that initialize successfully, the NULL component and one user defined component.  I can change this to say 1 non-null component if that reads better (or completely ignore the NULL component, and say "only 1 type of accelerator is supported")

Reply:
This error message is explicitly for when more than 1 (non-NULL) component is found (for example, if there are AMD and Nvidia cards in a single box, with no mca parameters).  I don't see why you couldn't have a box with AMD, Nvidia, and Ze cards in it?

Reply:
Consumer level cards? Maybe. For the more high-end cards it will probably not make sense however, since you will not be able to take advantage of the GPU interconnect (e.g. Nvlink, XGMI) between cards of different vendors, and they are a big reason for why you can scale to multiple GPUs per node for many jobs.

Reply:
Updated selection logic to make it simple for the user to figure out their bug.  New help text:
```
[Multiple Accelerators Found]
ERROR: Open MPI only supports one accelerator framework per node. Please pick one of the
initialized accelerator components via command line argument "--mca accelerator <component>".
The following accelerators initialized successfully: %s.
The program will abort to allow you to resolve the accelerator conflict.
```